e2e/tests/localAuthorityMaintainsCaseAfterSDO_test.js

```diff
@@ -0,0 +1,61 @@
+const config = require('../config.js');
+const hearingDetails = require('../fixtures/hearingTypeDetails.js');
+const directions = require('../fixtures/directions.js');
+
+let caseId;
+
+Feature('draft CMO after sdo');
```

Comment:
Feature name should be broader than this no? Could we re-word this to reflect the case state rather than an event.

Reply:
changed as suggested